<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RAMSIS</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="projectSummaryLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Project Summary</string>
    </property>
   </widget>
   <widget class="QLabel" name="projectNameLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>No Project Loaded</string>
    </property>
   </widget>
   <widget class="QLabel" name="startDateLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Starts: -</string>
    </property>
   </widget>
   <widget class="QLabel" name="endDateLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ends: -</string>
    </property>
   </widget>
   <widget class="QLabel" name="seismicEventCountLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0 Seismic Events</string>
    </property>
   </widget>
   <widget class="QLabel" name="exposureDataLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>No exposure data available</string>
    </property>
   </widget>
   <widget class="QLabel" name="boreholeDataLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>No borehole data available</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuOpen_Recent">
     <property name="title">
      <string>Open Recent</string>
     </property>
    </widget>
    <addaction name="actionNew_Project"/>
    <addaction name="actionOpen_Project"/>
    <addaction name="menuOpen_Recent"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionImport_Seismic_Data"/>
    <addaction name="actionImport_Hydraulic_Data"/>
    <addaction name="separator"/>
    <addaction name="actionView_Data"/>
   </widget>
   <widget class="QMenu" name="menuSimulation">
    <property name="title">
     <string>Simulation</string>
    </property>
    <addaction name="actionStart_Simulation"/>
    <addaction name="actionPause_Simulation"/>
    <addaction name="actionStop_Simulation"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionShow_3D"/>
    <addaction name="actionForecasts"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuSimulation"/>
   <addaction name="menuWindow"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionScenario"/>
   <addaction name="separator"/>
   <addaction name="actionSimulation"/>
   <addaction name="actionForecasts"/>
   <addaction name="actionShow_3D"/>
   <addaction name="actionTimeline"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionImport_Seismic_Data">
   <property name="text">
    <string>&amp;Import Seismic Data...</string>
   </property>
  </action>
  <action name="actionView_Data">
   <property name="text">
    <string>View Data</string>
   </property>
  </action>
  <action name="actionStart_Simulation">
   <property name="text">
    <string>Start Simulation</string>
   </property>
  </action>
  <action name="actionPause_Simulation">
   <property name="text">
    <string>Pause Simulation</string>
   </property>
  </action>
  <action name="actionStop_Simulation">
   <property name="text">
    <string>Stop Simulation</string>
   </property>
  </action>
  <action name="actionShow_3D">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/toolbar-buttons/images/view-3d-toolbar-button.png</normaloff>:/toolbar-buttons/images/view-3d-toolbar-button.png</iconset>
   </property>
   <property name="text">
    <string>3D View</string>
   </property>
  </action>
  <action name="actionImport_Hydraulic_Data">
   <property name="text">
    <string>Import Hydraulic Data...</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="text">
    <string>&amp;Open Project...</string>
   </property>
  </action>
  <action name="actionNew_Project">
   <property name="text">
    <string>New Project...</string>
   </property>
  </action>
  <action name="actionForecasts">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/toolbar-buttons/images/forecast-window-toolbar-button.png</normaloff>:/toolbar-buttons/images/forecast-window-toolbar-button.png</iconset>
   </property>
   <property name="text">
    <string>Forecasts</string>
   </property>
  </action>
  <action name="actionSimulation">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/toolbar-buttons/images/simulate-toolbar-button.png</normaloff>:/toolbar-buttons/images/simulate-toolbar-button.png</iconset>
   </property>
   <property name="text">
    <string>Simulation</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
  <action name="actionTimeline">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/toolbar-buttons/images/timeline-window-toolbar-button.png</normaloff>:/toolbar-buttons/images/timeline-window-toolbar-button.png</iconset>
   </property>
   <property name="text">
    <string>Timeline</string>
   </property>
   <property name="toolTip">
    <string>Show Timeline</string>
   </property>
  </action>
  <action name="actionScenario">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/toolbar-buttons/images/scenario-toolbar-button.png</normaloff>:/toolbar-buttons/images/scenario-toolbar-button.png</iconset>
   </property>
   <property name="text">
    <string>Scenario</string>
   </property>
   <property name="toolTip">
    <string>Compute Scenario</string>
   </property>
  </action>
  <action name="actionImport_FDSNWS_Data">
   <property name="text">
    <string>Import FDSNWS Data</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
